UPGRADE-6.3.md

```diff
@@ -182,6 +182,13 @@ The `backtrace_enabled` and `backtrace_limit` keys
 were moved under the `database_profiler` key (e.g. `chameleon_system_debug: database_profiler: backtrace_enabled`
 instead of `chameleon_system_debug: backtrace_enabled`). Existing configuration should be adjusted.
 
+## Access denied page
+
+The access denied page should now support an additionally mapped parameter `loginFormAction` as a target action for the shown 
+login form. This avoids the second unexpected AccessDeniedException.
```

Comment:
I think this second exception is not well-known for developers and we should explain it. How about this: "This avoids an unexpected AccessDeniedException on login submit caused by submitting the form to the wrong URL."

Still this suggestion would not express that login worked by accident - not sure if that is more confusing than helpful.

Reply:
Message changed:
This avoids another and then unexpected AccessDeniedException when logging in from the access-denied page.
Login still worked in that case however.